cpp/data/system_updates/159.vufind.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `vufind`.`tuefind_rss_feeds` 
+ADD COLUMN `type` SET('feed', 'news', 'calendar') NOT NULL DEFAULT 'news' AFTER `subsystem_types`;
```

Comment:
- Instead of SET we should use ENUM
- Allowed values should only be 'news' or 'calendar', not 'feed'